tempto-examples/src/main/resources/tempto-configuration.yaml

```diff
@@ -77,6 +77,8 @@ databases:
 
 tests:
   fs:
+    type: hdfs
+    prefix: ""
```

Comment:
do I need to specify this if it is empty?

Reply:
Good catch.  This can probably be written that if it is not defined then it will default to "".